tools/doc/type-parser.js

```diff
@@ -128,7 +135,10 @@ function toLink(typeInput) {
         typeLinks.push(
           `<a href="${typeUrl}" class="type">&lt;${typeTextFull}&gt;</a>`);
       } else {
-        typeLinks.push(`<span class="type">&lt;${typeTextFull}&gt;</span>`);
+        throw new Error(
+          `Unrecognized type: '${typeTextFull
+          }'. Please, edit the type or update the 'tools/doc/type-parser.js'.`
```

Comment:
Please concat the string instead of using a multi line template string.

Reply:
Done in the fixup comment.